Felienne/hedy
# [BUGFIX] Fix debugger on codeshare page

**Description**
The debugger (🐞 button) does not work when executing shared code. This PR fixes that by removing the need of receiving a specific level number. 

**Fixes #2787**

**How to test**
Create a Hedy program (with at least 2 lines). Save & share the code. Visit the code's URL. Try to execute the program using the debugger. It should be able to execute the program line-by-line.
